$NetBSD: patch-CMakeLists.txt,v 1.9 2025/09/30 10:44:39 adam Exp $

Don't look for boost_system library.
Avoid application bundle on macOS.

--- CMakeLists.txt.orig	2024-11-19 15:51:02.000000000 +0000
+++ CMakeLists.txt
@@ -373,7 +373,7 @@ IF(WIN32)
   ENDIF()
 ENDIF(WIN32)
 
-FIND_PACKAGE(Boost 1.47 COMPONENTS filesystem system)
+FIND_PACKAGE(Boost 1.47 COMPONENTS filesystem)
 
 IF (NOT Boost_FOUND)
   IF(WIN32)
@@ -587,16 +587,6 @@ IF (APPLE)
   ENDIF (NOT MAC_SELF_CONTAINED_BUNDLE)
 ENDIF (APPLE)
 
-IF (HUGIN_SHARED AND UNIX AND NOT APPLE)
-  SET(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/${LIBDIR}/hugin")
-  SET(HUGIN_LIBS_PRIVATE_DIR "1" CACHE INTERNAL "Store internal
-    libraries in private subdirectory" )
-ELSE ()
-  # On OS X files are moved around after make install, do not try to use
-  # rpath or CMAKE_INSTALL_NAME_DIR
-  SET(HUGIN_LIBS_PRIVATE_DIR "0" CACHE INTERNAL "Store internal
-    libraries in private subdirectory" )
-ENDIF()
 
 IF (UNIX)
   IF (HUGIN_SHARED)
